### DISCUSSION TITLE
Is it possible to set specific temperature vs simply select modes - away or not away?

### DISCUSSION THREAD

**Discussion by WhatevsJJ** [PeteRager/lennoxs30#316] [Q&A]
For example, when the wife leaves i would like to set heat to 64deg, when we are both gone I would like to set it to 55deg.
Thanks for this awesome toolset!

**Comment by PeteRager** [comment]
The Lennox panel does not have that capability.  

The way you would do this is via an Automation and use the climate.set_temperature service.  You'd need device trackers for both of your phones and use those as triggers.

**Comment by WhatevsJJ** [comment]
Ah, I have a couple of automations configured, one is below, that simply sets systems to away.  I hate to ask, but where/how would I use the climate.set_temperature service in this example?

 

Thank you for any help you can provide,Jason

 

alias: HVAC - Normal Mode - Jill joins MyWiFi

description: ""

trigger:

  - platform: state

    entity_id:

      - device_tracker.lonestar_jill_phone_ping_40_103

    from: not_home

    to: home

    for:

      hours: 0

      minutes: 2

      seconds: 0

condition: []

action:

  - type: turn_off

    device_id: e068ce96df12eb88ee4457c39ef3344a

    entity_id: f632c89a92d4c423e06c004936d080d4

    domain: switch

  - device_id: ee3c48d828856ecdce22221b7b7d9015

    domain: climate

    entity_id: 82216f16f6236f2f06663d4573869db6

    type: set_preset_mode

    preset_mode: none

mode: single

**Comment by PeteRager** [comment]
It looks like you are using presets.  So you could add two new presets on the panel for the two modes you need (wife away, both away) and use the automation to set the presets. You'll may need to reload the integration for the new presets to populate.  

Typically you'd use set_temperature when HA is controlling your HVAC schedule.  In your case the presets are in charge of the schedule and you are using HA to select presets.
